server/src/server.ts

```diff
@@ -178,11 +178,20 @@ connection.onCompletion(async function(
 });
 
 function getWorkspacePathToFile(doc: {uri: string}): string|undefined {
-  const match = doc.uri.match(/^file:\/\/(.*)/);
+  const match = doc.uri.match(/^file:\/\/\/?(.*)/);

```

Comment:
I'd love comments on regexes to explain what they're trying to match. Here I'd want to highlight the trailing slash, perhaps.